File: graph01/bank_end/ruoyi-system/src/main/java/com/ruoyi/system/domain/Favorite.java
package com.ruoyi.system.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * ClassName: Favorite
 * Package: com.ruoyi.system.domain
 * Description:
 *
 * @Author 张鹏
 * @Create 2026/4/3 23:01
 * @Version 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Favorite {
    private Integer id;
    private Integer favoriteId;
    private Integer typeId;
    private Integer userId;
}

File: graph01/graph_fontend/vo/Favorite.js
/**
 * Favorite VO - 收藏实体
 */
export class Favorite {
  constructor(data) {
    this.id = data.id || null;
    this.favoriteId = data.favoriteId || null;
    this.typeId = data.typeId || null;
    this.userId = data.userId || null;
  }
}
